utils/eventtracking.js

```diff
@@ -3,26 +3,44 @@ const mixpanel = require('mixpanel').init(MIXPANEL_TOKEN);
 
 const uuid = require('uuid');
 const chalk = require('chalk');  // colorize output
+const crypto = require('crypto');
 const readline = require('readline');
 const settings = require('./settings');
 
 const TRACKING_ENABLED_KEY = 'trackingEnabled';
 const TRACKING_SESSION_ID_KEY = 'trackingSessionId';
 
+const isGitPod = () => {
+    return !!process.env.GITPOD_WORKSPACE_URL;
+};
+
+const getGitPodUserHash = () => {
+    if (!process.env.GITPOD_GIT_USER_NAME) {
+        return null;
+    }
+    return crypto.createHash('sha256').update(process.env.GITPOD_GIT_USER_NAME, 'utf8').digest('hex').toString();
```

Comment:
Cool that we can track sessions across users in anonymized way. Wonder if it's better to use `GITPOD_GIT_USER_EMAIL` as my understanding is that `GITPOD_GIT_USER_NAME` is full name (so not necessarily unique).

Reply:
sure, updated.